crates/sui-keys/src/keystore.rs

```diff
@@ -233,6 +245,39 @@ impl AccountKeystore for FileBasedKeystore {
         }
     }
 
+    /// Updates an old alias to the new alias
+    fn update_alias(
```

Comment:
Am I right in thinking that these two implementations are identical and only rely on other functions in the trait? If so, you could make it a default implementation in the trait.